code_schemes/all_locations_s01e05.json

```diff
@@ -3,6 +3,94 @@
   "Name": "All Locations S01E05",
   "Version": "0.0.0.1",
   "Codes": [
+    {
+      "CodeID": "code-145fd9ba",
+      "CodeType": "Normal",
+      "DisplayText": "educate the community",
+      "NumericValue": 1,
+      "StringValue": "educate_the_community",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-cb631b67",
+      "CodeType": "Normal",
+      "DisplayText": "follow vet/doctor's prescription",
+      "NumericValue": 2,
+      "StringValue": "follow_vet_or_doctors_prescription",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-4d466ac5",
+      "CodeType": "Normal",
+      "DisplayText": "questions and narratives",
+      "NumericValue": 3,
+      "StringValue": "questions_and_narratives",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-89500909",
+      "CodeType": "Normal",
+      "DisplayText": "maintain hygiene",
+      "NumericValue": 4,
+      "StringValue": "maintain_hygiene",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-8717dbf9",
+      "CodeType": "Normal",
+      "DisplayText": "enact strict laws",
+      "NumericValue": 5,
+      "StringValue": "enact_strict_laws",
+      "VisibleInCoda": true
+    },
+     {
+      "CodeID": "code-9f9b75c6",
+      "CodeType": "Normal",
+      "DisplayText": "avoid over the counter drugs",
+      "NumericValue": 6,
+      "StringValue": "avoid_over_the_counter_drugs",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-fc72cebd",
+      "CodeType": "Normal",
+      "DisplayText": "proper animal feeding",
+      "NumericValue": 7,
+      "StringValue": "proper_animal_feeding",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-588a0dc3",
+      "CodeType": "Normal",
+      "DisplayText": "deploying more veterinary officers",
+      "NumericValue": 8,
+      "StringValue": "deploying_more_veterinary_officers",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-15590006",
+      "CodeType": "Normal",
+      "DisplayText": "vaccinate animals",
+      "NumericValue": 9,
+      "StringValue": "vaccinate_animals",
+      "VisibleInCoda": true
+    },
+     {
+      "CodeID": "code-b3a4a65a",
+      "CodeType": "Normal",
+      "DisplayText": "do not consume product from sick animals",
+      "NumericValue": 10,
```

Comment:
This has a different numeric value in the the kilifi dataset, I think these should be the same because the code is the same.

Reply:
This may not be possible because Bungoma has more codes and the NumericValues have already been used there.

Reply:
I don't understand - if the numeric value has already been used in bungoma then the numeric values there should need updating too. All the other datasets should be able to use a subset of exactly the same codes as are in the all dataset

Reply:
Okay, I think this depends on whether we want to keep the NumericValues sequential/in order in all schemes or apply the same value across the schemes. If the values are different does it affect the label tagging in the pipeline?